web/styles/innov-footer.scss

```diff
@@ -0,0 +1,89 @@
+.footerLinkCustom {
```

Comment:
let's try to keep consistent on just using kebab case for classes (class-names-with-hyphens)